pkg/cmd/project/project.go

```diff
@@ -29,7 +29,14 @@ func NewCmdProject(f *cmdutil.Factory) *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "project <command>",
 		Short: "Work with GitHub Projects.",
-		Long:  "Work with GitHub Projects. Note that the token you are using must have 'project' scope, which is not set by default. You can verify your token scope by running 'gh auth status' and add the project scope by running 'gh auth refresh -s project'.",
+		Long: heredoc.Docf(`
+			Work with GitHub Projects.
+
+			Note that the token you are using must have 'project' scope,
```

Comment:
```suggestion
			Note that the token you are using must have %[1]sproject%[1]s scope,
```

Bringing this in line with `gh auth login` doc behavior:

https://github.com/cli/cli/blob/69fff52026428c0130d812037573b5c9cc0e77bb/pkg/cmd/auth/login/login.go#L72